app/modules/clamav-container-instance/container.tf

```diff
@@ -0,0 +1,38 @@
+resource "azurerm_container_group" "back_office_containers" {
+  #checkov:skip=CKV_AZURE_98: Ensure that Azure Container group is deployed into virtual network
+  name                = "back-office-containers-${var.resource_suffix}"
+  location            = var.location
+  resource_group_name = var.resource_group_name
+  ip_address_type     = "Private"
+  dns_name_label      = "clamav-endpoint"
+  os_type             = "Linux"
+  network_profile_id  = azurerm_network_profile.back_office_clamav.id
```

Comment:
network_profile_id is actually deprecated as of the 2021-07-01 API Version, but we're still using an old version of the Azure Terraform provider (3.6.0 (May 12, 2022). network_profile_id was deprecated in version 3.16.0.